tests/local.rs

```diff
@@ -35,6 +35,21 @@ async fn find_and_click_link(mut c: Client, port: u16) -> Result<(), error::CmdE
     c.close().await
 }
 
+async fn serialize_element(mut c: Client, port: u16) -> Result<(), error::CmdError> {
+    let url = sample_page_url(port);
+    c.goto(&url).await?;
+    let elem = c.find(Locator::Css("#other_page_id")).await?;
+
+    // Check that webdriver understands it
+    c.execute(
+        "arguments[0].scrollIntoView(true);",
+        vec![serde_json::to_value(elem)?],
+    )
+    .await?;
```

Comment:
Does this actually test that the serialization works correctly? Just as a sanity check, could you check that passing in some other random string here and see that the test indeed fails? I'm not sure whether the exception in the JavaScript code actually bubbles up to the WebDriver.

Reply:
Yes, it really tests it. I manually provided an incorrect element ID instead and got an error.

I could have left it in the code though. I will do it shortly.